docs/api-docs/catalog/v2-vs-v3.md

```diff
@@ -0,0 +1,128 @@
+* You can include sub-resources within a request.
+* Each V3 resource includes a meta object, simplifying pagination.
+* V3 [Brands](https://developer.bigcommerce.com/api-reference/store-management/catalog/brands/getbrandbyid), [Categories](https://developer.bigcommerce.com/api-reference/store-management/catalog/category/getcategorybyid), [Products](https://developer.bigcommerce.com/api-reference/store-management/catalog/products/getproductbyid), and [Product Variants](https://developer.bigcommerce.com/api-reference/store-management/catalog/product-variants/getvariantbyid) expose a metafields resource for use by developers to store custom data.
+* V3 API is optimized for performance. In general, data can be sent, received, and processed faster via V3, relative to V2.
+
+## Difference between V2 and V3 Catalog APIs
+
+### Product variants and modifiers instead of option sets
+
+V3 Catalog introduced a new option model which included two types of options: [variant options](https://developer.bigcommerce.com/api-docs/store-management/products-overview#variant-options) and [modifiers](https://developer.bigcommerce.com/api-docs/store-management/products-overview#modifier-options). You can now create variants and modifiers in one call without having to create option sets beforehand. The new option model simplified the creation and management of variant prices and modifier adjusters and removed the need to use complex rules, in all but some cases.
+
+#### Variant options
+[Variant options](https://developer.bigcommerce.com/api-docs/store-management/products-overview#variant-options) represent options used for variant generation. A shopper has to choose from available variant options before adding a product to the cart. Variant options include multiple-choice types such as swatch, rectangle, radio button, and dropdown.
+
+Variant options example: size or color.
+
+#### Variants
+[Variants](https://developer.bigcommerce.com/api-docs/store-management/products-overview#variant) are created based on the combination of variant options and have their own attributes such as image, prices, weight, and stock level.
+
+Variant example: large blue t-shirt.
+
+#### Modifiers
+[Modifiers](https://developer.bigcommerce.com/api-docs/store-management/products-overview#modifier-options) represent options used for additional product customization such as gift wrapping, engraving, text to be printed on a t-shirt, or a warranty. Unlike variant options, modifiers do not generate variants and can be displayed as *required* or *optional* on the storefront. You can use modifiers along with [rules](https://developer.bigcommerce.com/api-docs/store-management/products-overview#complex-rules) to change a product's price or weight.
+Modifiers include multiple-choice types such as swatch, rectangle, radio button, and dropdown and non-multiple-choice types such as text, multi-line text, date picker, file upload, and pick list.
```

Comment:
Modifiers include multiple-choice types such as swatch, rectangle, radio button, dropdown, and non-multiple-choice types such as text, multi-line text, date picker, file upload, and pick list.